notify_linux.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package caddy
```

Comment:
Don't you need a `// +build linux` up here?

Reply:
It is covered by the filename suffix, isn't it?

Reply:
Ah, TIL. I didn't realize filename suffixes were part of that.

Reply:
I finally found the official documentation for this… It is mentioned in a very tiny section here:

https://golang.org/cmd/go/#hdr-Build_constraints

```
 If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:

*_GOOS
*_GOARCH
*_GOOS_GOARCH
```